.github/workflows/resource-count-generator.yaml

```
@@ -0,0 +1,30 @@
+name: ResourceCountGenerator
```

```suggestion
name: ResourceCount
```

nit: Do we need to throw the word generator in there?